File: cattleshed-conf/templates/erlang/prog.erl
% This file is a "Hello, world!" in Erlang for wandbox.
-module(prog).

% Erlang for Wandbox must exports main/0
-export([main/0]).

main() ->
    io:format("Hello, Wandbox!~n").

% Erlang language reference:
%   http://erlang.org/doc/reference_manual/users_guide.html

File: cattleshed-conf/templates/r/prog.R
# This file is a "Hello, world!" in R language for wandbox.

data(iris)                               ## the "hello, world" of statistics.
summary(iris)                            ## a simple summary of the columns
fit <- lm(Sepal.Length ~ . , data=iris)  ## basic regression
summary(fit)                             ## summary

cat("All done\n")

# R language references:
#   https://cran.r-project.org/manuals.html
